src/carousel/carousel.ts

```diff
@@ -87,7 +104,9 @@ export class NgbCarousel implements AfterContentChecked,
   private _destroy$ = new Subject<void>();
   private _interval$ = new BehaviorSubject(0);
   private _mouseHover$ = new BehaviorSubject(false);
+  private _focused$ = new BehaviorSubject(false);
   private _pauseOnHover$ = new BehaviorSubject(false);
+  private _pauseOnFocus$ = new BehaviorSubject(false);
```

Comment:
I think I would turn the default value to be `true`.

Reply:
Actually, the default value is not set here but in `carousel-config.ts`.
I just did like `_pauseOnHover$` which is also `true` by default.

Reply:
But then, it means all our `BehaviorSubject` should be initialised in the constructor with the proper initial value coming from the config no ?

Reply:
Actually, currently we do the following in the constructor:
```js
    this.interval = config.interval;
    this.wrap = config.wrap;
    this.keyboard = config.keyboard;
    this.pauseOnHover = config.pauseOnHover;
    this.pauseOnFocus = config.pauseOnFocus;
    this.showNavigationArrows = config.showNavigationArrows;
    this.showNavigationIndicators = config.showNavigationIndicators;
```
So those `BehaviorSubject` objects are created with an arbitrary value and then updated in the constructor (through the setters) with the correct default value, so the behavior is correct, even though we could improve a bit the performance by directly creating those objects in the constructor (I am not sure it would make a noticeable difference, though).

Reply:
Ok, then let's keep it as it is today!